### ISSUE TITLE
While optimizing the frozen graph to IE using export.py in LPR below error occurs:[ ERROR ]  Cannot infer shapes or values for node "CTCGreedyDecoder". [ ERROR ]  index 1 is out of bounds for axis 0 with size 1 [ ERROR ]   [ ERROR ]  It can happen due to bug in custom shape infer function <function CTCGreedyDecoderOp.ctc_greedy_decoder_infer at 0x7f1bed43ec80>. [ ERROR ]  Or because the node inputs have incorrect values/shapes. [ ERROR ]  Or because input shapes are incorrect (embedded to the model or passed via --input_shape).

### ISSUE THREAD

**Issue by PravyAI** [open-edge-platform/geti#33]
- [x] I have searched the [issues](https://github.com/opencv/training_toolbox_tensorflow/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
    If you're describing a bug, tell us what should happen.
    If you're suggesting a change/improvement, tell us how it should work.

## Current Behavior
    If describing a bug, tell us what happens instead of the expected behavior.
    If suggesting a change/improvement, explain the current behavior.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Context
    How has this issue affected you? What are you trying to accomplish?
    Providing context helps us come up with a solution that is most useful in the real world.

## Your Environment

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):
```
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
```
- Python version:
- CUDA/cuDNN version:
- GPU model and memory:

    Include as many relevant details about the environment with which you experienced the bug.

**Comment by alexey-trushkov** [comment]
@PravyAI could you please provide more information how to reproduce this issue. Please follow template for filing the bug.

**Comment by PravyAI** [comment]
In LPR repo i executed following steps:-

1) Download training data and extract it in data/synthetic_chinese_license_plates folder.

2) python data/synthetic_chinese_license_plates/make_train_val_split.py <path to annotation>

3)  python3 train.py chinese_lp/config.py

**### Now after completion of training , i did below steps:**

4> python lpr/export.py lpr/config.py /opt/intel/computer_vision_sdk_2018.4.420/deployment_tools/model_optimizer/mo.py

**### Expected: Bin and XML file.**

**### But i got below error:**
Common parameters:
	- Path to the Input Model: 	/home/prabhat/Desktop/Numberplate1/training_toolbox/lpr/model/ie_model/graph.pb.frozen
	- Path for generated IR: 	/home/prabhat/Desktop/Numberplate1/training_toolbox/lpr/model/ie_model
	- IR output name: 	/home/prabhat/Desktop/Numberplate1/training_toolbox/lpr/model/ie_model/graph
	- Log level: 	ERROR
	- Batch: 	Not specified, inherited from the model
	- Input layers: 	input
	- Output layers: 	d_predictions
	- Input shapes: 	[1,24,94,3]
	- Mean values: 	Not specified
	- Scale values: 	Not specified
	- Scale factor: 	255.0
	- Precision of IR: 	FP32
	- Enable fusing: 	True
	- Enable grouped convolutions fusing: 	True
	- Move mean values to preprocess section: 	False
	- Reverse input channels: 	True
TensorFlow specific parameters:
	- Input model in text protobuf format: 	False
	- Offload unsupported operations: 	False
	- Path to model dump for TensorBoard: 	None
	- List of shared libraries with TensorFlow custom layers implementation: 	None
	- Update the configuration file with input/output node names: 	None
	- Use configuration file used to generate the model with Object Detection API: 	None
	- Operations to offload: 	None
	- Patterns to offload: 	None
	- Use the config file: 	None
Model Optimizer version: 	1.4.292.6ef7232d
[ ERROR ]  Cannot infer shapes or values for node "CTCGreedyDecoder".
[ ERROR ]  index 1 is out of bounds for axis 0 with size 1
[ ERROR ]  
[ ERROR ]  It can happen due to bug in custom shape infer function <function CTCGreedyDecoderOp.ctc_greedy_decoder_infer at 0x7f4a9a364f28>.
[ ERROR ]  Or because the node inputs have incorrect values/shapes.
[ ERROR ]  Or because input shapes are incorrect (embedded to the model or passed via --input_shape).
[ ERROR ]  Run Model Optimizer with --log_level=DEBUG for more information.
[ ERROR ]  Stopped shape/value propagation at "CTCGreedyDecoder" node. 
 For more information please refer to Model Optimizer FAQ (<INSTALL_DIR>/deployment_tools/documentation/docs/MO_FAQ.html), question #38.

**Comment by PravyAI** [comment]
@alexey-trushkov  Issue resolved by using OpenVino 2018 R5. Thanks
